bin/license/helper_funcs.sh

```
@@ -0,0 +1,92 @@
+		if [ "$project" != "null" ]; then
+			check_license_valid $project
+		fi
+	done
+}
+
+check_license_valid()
+{
+	project=$1
+	RECORD_NAME=$(dx find data --class record --path $project --name Sentieon_License --brief|head -1)
+	RECORD=$(dx describe $RECORD_NAME --json)
+	if [ $(date -d $(echo $RECORD |jq .details.license_expiration|sed 's|"||g') +"%Y%m%d") -ge $(date +"%Y%m%d") ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)